src/FSharp.Stats/Array.fs

```diff
@@ -507,3 +507,19 @@ module Array =
         array 
         |> Array.filter (System.Double.IsNaN >> not)
 
+
+type Array() =
+
+    /// Creates an float array from an given interval.
```

Comment:
I would suggest to use actual xml documentation, that way we can benefit from the power of fsdocs api docs generation. In this case i would change the docs to the following:

```fsharp
/// <summary>
/// Creates an float array from an given interval.
/// </summary>
/// <param name="start">start value (is included)</param>
/// <param name="stop">end value (by default is included )</param>
/// <param name="Num">sets the number of elements in the array. If not set, stepsize = 1.</param>
/// <param name="IncludeEndpoint">If false, array does not contain stop value</param>
static member linspace(start:float,stop:float,?Num:int,?IncludeEndpoint:bool) : float [] = 
...
```

I would try to use this documentation style for every new addition, and maybe slowly adding it to old stuff when adequate.

### Why?

just take a look at what can be auto generated from that here: https://plotly.net/reference/plotly-net-chart.html

You also have the additional benefit of IDEs showing the param hints as tooltips:
![image](https://user-images.githubusercontent.com/21338071/223514002-f9219ebe-66d2-4a62-adc6-06ffba4e36fc.png)